precompiles/staking/StakingI.sol

```diff
@@ -142,6 +148,21 @@ interface StakingI is authorization.AuthorizationI {
         uint256 value
     ) external returns (bool success);
 
+    /// @dev Defines a method for edit a validator.
+    /// @param description Description parameter to be updated, an empty string ""
+    /// means keeping it as is and not updating it.
```

Comment:
This is not true. The current implementation is not handling the case in which the description remains the same. If we want to provide the same functionality of the Cosmos SDK module we have to properly manage this case